environments/auth-with-multi-alias/README.md

```diff
@@ -0,0 +1,25 @@
+# Auth with Multiple Sign In Aliases
+
+This environment allows users to sign in using a username, email, or phone number. Allowing users to sign in using multiple aliases requires configuring your `auth` backend using the Amplify CLI.
+
+## Configuring the Backend
+
+To configure your application's `auth` backend to allow multiple sign in aliases, follow these steps:
+
+1. In your application's root directory, add authentication using the Amplify CLI:
+
+   ```shell
+   amplify add auth
+   ```
+
+1. When prompted `Do you want to use the default authentication and security configuration?`, select `Manual configuration`
+1. When prompted `How do you want users to be able to sign in?`, select the aliases you would like users to be able to use to sign in to your application
+   ![](multi-alias-config.png)
+
+   _Note: a username is still required when a user creates an account and is treated as a sign in alias regardless of if it was selected as an option._
```

Comment:
In your image, only `username` and `phone_number` are selected.

Does that mean that the `username` value === `email`?  Or that, just like your demo, the backend will return `['username', 'email', 'phone_number']`?

Reply:
Even without `username` selected, it's still treated as a valid login mechanism and is required for creating an account. It's a bit strange, but one may think of it as `username` always being a selected option.